### PR TITLE
Ticket 10: POST responds with username and comment

### PR DESCRIPTION
**Pull request by MCodes9**

### CODE REVIEW

**Review by scar1377 — APPROVED** [review]
Looks good to merge. The only thing may need to change is this endpoint is mostly about comments instead of articles, so would be better to put the functions into comments controllers and models. Please make the change and go ahead to merge it to main

File: controllers/errors.controller.js [hunk 1]
```diff
@@ -3,8 +3,13 @@ exports.handlePathErrors = (req, res) => {
 };
 
 exports.handlePSQLErrors = (err, req, res, next) => {
+  console.log(err);
```
Comment (leekli) [inline controllers/errors.controller.js]: Just need remove this console log :)

Comment (MCodes9) [reply controllers/errors.controller.js]: Done! @leekli @scar1377 
Checked tests, all still working. 
Merging branches now :)

**Review by leekli — APPROVED** [review]
Good to merge.

**Pull request merged**